drivers/segger/sysview.c

```diff
@@ -333,16 +234,12 @@ void sched_note_resume(FAR struct tcb_s *tcb)
 #endif
 
 #ifdef CONFIG_SCHED_INSTRUMENTATION_IRQHANDLER
-void sched_note_irqhandler(int irq, FAR void *handler, bool enter)
+static void note_sysview_irqhandler(FAR struct note_driver_s *drv, int irq,
+                                    FAR void *handler, bool enter)
 {
-  if (!sysview_isenabled_irq(irq, enter))
```

Comment:
let's cast drv to avoid using the global variable(g_note_sysview_driver) as much as possible:
FAR struct note_sysview_driver_s *sysview = ...

Reply:
done